openedx/core/djangoapps/user_authn/utils.py

```diff
@@ -67,3 +67,10 @@ def is_registration_api_v1(request):
     :return: Bool
     """
     return 'v1' in request.get_full_path() and 'register' not in request.get_full_path()
+
+
+def should_redirect_to_logistration_mircrofrontend():
+    """
+    Checks if login/registration should be done via MFE.
+    """
+    return settings.FEATURES.get('ENABLE_LOGISTRATION_MICROFRONTEND')
```

Comment:
Do we need to use the `site_configuration` here to get the settings value?

Reply:
I think these are overridden in edx-internal and from what I know these are fetched using settings.
https://github.com/edx/edx-internal/blob/master/edx-remote-config/prod/lms.yml